script/release/release-artifact-cleanup.js

```diff
@@ -85,14 +84,14 @@ async function cleanReleaseArtifacts () {
 
   if (releaseId) {
     if (isNightly) {
-      const deletedNightlyDraft = await deleteDraft(releaseId, 'nightlies')
+      try {
+        await deleteDraft(releaseId, 'nightlies')
 
-      // don't delete tag unless draft deleted successfully
-      if (deletedNightlyDraft) {
-        await Promise.all([
-          deleteTag(args.tag, 'electron'),
-          deleteTag(args.tag, 'nightlies')
-        ])
+        // We only need to delete the Electron tag since the
+        // nightly tag is only created at publish-time.
+        await deleteTag(args.tag, 'electron')
+      } catch (err) {
+        console.log(`${fail} Couldn't complete cleanup for nightly: `, err)
```

Comment:
I'm not sure this error handling will work as intended?

Neither `deleteDraft()` nor `deleteTag()` will throw errors because they wrap everything in internal try-catch blocks and return a success flag, so I think (1) `deleteTag()` will be called even if `deleteDraft()` fails and (2) the `catch` clause will never be reached

Reply:
For example, previously we didn't try to delete tags if `deleteDraft()` failed, but the new code would try to delete the tag `'electron'` tag anyway

Reply:
that's correct @ckerr 
> new code would try to delete the tag 'electron' tag anyway

that's is what's intended!